presto-product-tests/bin/locations.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
@electrum fixed, both for locations.sh and compose-commons.sh. Anything else? :)